autoscheduler/frontend/src/components/SchedulingPage/CourseSelectColumn/CourseSelectCard/ExpandedCourseCard/BasicSelect/BasicSelect.css

```diff
@@ -17,4 +17,5 @@
 .gray-text {
     color: rgba(0, 0, 0, 0.56);
     padding-bottom: 8px;
+    padding-right: 0.5vw;
```

Comment:
I think this would probably make more sense to have it be `8px` to have it match `padding-bottom`. In general I typically like to have the padding/margin be constants